kolibri/core/assets/src/views/ExamReport/CurrentTryOverview.vue

```diff
@@ -128,9 +129,15 @@
         type: String,
         default: '',
       },
+      // Whether the modality is a Survey - conditionalizes styles
+      isSurvey: {
+        type: Boolean,
+        default: false,
+      },
     },
     computed: {
       ...mapGetters(['currentUserId']),
+      // QUESTION: Why? Wouldn't anything else fail validation? Why not just call tryValidator here?
```

Comment:
Validators don't run in production, only in development. This value can still be undefined, so we need to handle that case. Could swap it out for `tryValidator` instead, which might be more robust.

Reply:
If the component defines a prop as required and validates it then I think it should be able to operate under the assumption that it won't be used incorrectly. The parent component should (and I think in all/most cases do - can confirm in a bit) guard rendering CurrentTryOverview w/ v-if

Reply:
I don't disagree - but it's not how Vue handles it, so better to be defensive, I think.